tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -82,6 +82,24 @@ def testValidateBadEntityNamespace(self):
     if instance.IsValidEntityInstance():
       self.fail('exception not raised')
 
+
+  def testValidateBadAbstractType(self):
+    parsed = instance_parser.parse_yaml(
```

Comment:
this test is brittle because it uses a real abstract type that could be deleted.  A better version would define both the instance to check and the type to check against, however since we have to rewrite a bunch of this it's not a blocker right now.

Reply:
sgtm